Software/res/moodlamps/README.md

```diff
@@ -0,0 +1,144 @@
+Moodlamp scripting
+---------
+Prismatik uses [QJSEngine](https://doc.qt.io/qt-5/qjsengine.html) for parsing and running moodlamp scripts, so if you are familiar with JavaScript you'll be fine.
+
+The scripts live in your user's Prismatik folder:
+- Windows: `C:\Users\<you>\Prismatik\moodlamps`
+- macOS & Linux: `~/.Prismatik/moodlamps`
+
+File name should obey these rules:
+- allowed characters `a-z`, `0-9` and `_` (underscore)
+- `.mjs` extension
+- unique name
+
+For ex: `my_cool_lamp.mjs`
+
+
+**The bare minimum template:**
+```js
+export const name = "your lamp name here"
+export const interval = 50 // frame time in ms
+
+/**
+ * @param {integer}	baseColor	integer RGB value of the base color that you set in Prismatik
+ * @param {array}	colors		array of RGB integers representing your LEDs
+ * @return {array}	array of new RGB integers representing your LEDs
+ */
+export function shine(baseColor, colors)
```

Comment:
why `shine`? Given it's executed every interval, I'd have picked something like `nextColor` or `tick`.

Reply:
just because that's what lamps do and it's the first thing that came to mind
I could live with `tick` yeah